scipy/signal/spectral.py

```diff
@@ -141,7 +141,7 @@ def periodogram(x, fs=1.0, window=None, nfft=None, detrend='constant',
                  scaling, axis)
 
 
-def welch(x, fs=1.0, window='hann', nperseg=256, noverlap=None, nfft=None,
+def welch(x, fs=1.0, window='hann', nperseg=None, noverlap=None, nfft=None,
```

Comment:
What I mean is that the docstring should reflect the true default in the function definition, and go on to explain the behavior. Explicit is better than implicit.